code/backend/batch/utilities/helpers/EnvHelper.py

```diff
@@ -164,6 +170,9 @@ def is_chat_model(self):
             return True
         return False
 
+    def get_env_var_bool(self, var_name: str, default: str = "True") -> bool:
```

Comment:
Is there a simple unit test that we can add around this - (glad you pulled this into a function btw!) :-)

Reply:
I think this is covered by the existing unit tests.